config/locales/server.en.yml

```diff
@@ -723,6 +723,7 @@ en:
         bio_raw: "About Me"
       user:
         ip_address: ""
+        password: "Password"
```

Comment:
This is what makes the "password" attribute translateable for the "is blah blah" password validation errors for full_messages